lib/metrician/configuration.rb

```diff
@@ -7,29 +7,49 @@ class Configuration
     def self.load
       reset_dependents
 
-      if env_location
-        # this should never raise unless a bad ENV setting has been set
-        raise(FileMissing.new(env_location)) unless File.exist?(env_location)
-        return YAML.load(env_location)
+      config = {}
+      config_locations.each do |location|
+        config = merge(config, YAML.load_file(location)) if File.exist?(location)
       end
+      config
+    end
 
-      if File.exist?(app_location)
-        return YAML.load_file(app_location)
+    def self.merge(a_config, b_config)
```

Comment:
Mostly active support is well-trod, so directly copying their code is usually where I go when I need one of their funcs. :)